### PR TITLE
Add new staging repo: cri-api

### PR DESCRIPTION
**Pull request by nikhita**
Ref:

- Repo creation request: https://github.com/kubernetes/org/issues/642
- Repo with initial empty commit: https://github.com/kubernetes/cri-api
- PR against test-infra for branch protection: https://github.com/kubernetes/test-infra/pull/11941
- PR for to add repo to staging against k/k: https://github.com/kubernetes/kubernetes/pull/75531
- PR for updating publishing-bot rules against k/k: https://github.com/kubernetes/kubernetes/pull/75531

/hold
This needs to be added to staging and the rules need to be updated

/assign @dims @sttts

### CODE REVIEW

File: hack/fetch-all-latest-and-push.sh [hunk 1]
```diff
@@ -43,6 +43,7 @@ repos=(
     cluster-bootstrap
     code-generator
     component-base
+    cri-api
```
Comment (sttts) [inline hack/fetch-all-latest-and-push.sh]: we should really make this consume the rules file. Or just move it over completely into the main publisher process. We maybe just lack a clone command. We fetches changes already.

**Comment by dims** [comment]
@nikhita the `/hold` can be cancelled?

**Comment by nikhita** [comment]
> @nikhita the /hold can be cancelled?

No, we can cancel the hold once the `cri-api` directory exists in staging. :)

**Comment by nikhita** [comment]
/hold cancel
https://github.com/kubernetes/kubernetes/pull/75531 was merged and rules have been updated.

File: hack/fetch-all-latest-and-push.sh (hunk 1 above)
Comment (nikhita) [reply hack/fetch-all-latest-and-push.sh]: Created https://github.com/kubernetes/publishing-bot/issues/171

**Comment by nikhita** [comment]
This needs a redeploy.

**Pull request merged**